gtsam/linear/SubgraphPreconditioner.cpp

```diff
@@ -149,20 +149,6 @@ void SubgraphPreconditioner::multiplyInPlace(const VectorValues& y, Errors& e) c
   Ab2_.multiplyInPlace(x, ei);                  // use iterator version
 }
 
-/* ************************************************************************* */
-// Apply operator A', A'*e = [I inv(R1')*A2']*e = e1 + inv(R1')*A2'*e2
-VectorValues SubgraphPreconditioner::operator^(const Errors& e) const {
```

Comment:
wait, this might be too much :-) SubgraphPreconditioner is not a Matrix, so why remove this ?

Reply:
Is using A^b for A^T * b notation in just not familiar with yet? I believe the thinking was this felt like operator abuse. I can only delete the ambiguous operator^'s if we'd prefer though.

Reply:
Yeah, I’d prefer if the iterative code is left as is.

I see your point, but there is code _outside_ GTSAM that implements other “Systems” that will break, and I have no time to revisit/sleuth the consequences at this time.